app/src/main/java/com/hunguet123/todoList/ui/delete/DeleteFragment.kt

```diff
@@ -0,0 +1,63 @@
+package com.hunguet123.todoList.ui.delete
+
+import android.os.Bundle
+import android.view.View
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.hunguet123.todoList.R
+import com.hunguet123.todoList.ui.home.HomeViewModel
+import kotlinx.android.synthetic.main.fragment_delete_item.*
+import org.koin.androidx.viewmodel.ext.android.viewModel
+
+class DeleteFragment : Fragment(R.layout.fragment_delete_item) {
+
+    private val viewModel : DeleteViewModel by viewModel()
+    private val homeViewModel : HomeViewModel by viewModel()
+
+    private var position = 0
+
+    var onPopBackstack: () -> Unit = {}
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        val bundle = this.arguments
+        if (bundle != null) {
+            position = bundle.getInt("key")
```

Comment:
Bản thân Delete không cần thiết phải tách màn hình mới, em chỉ cần show 1 dialog confirm 
Are u sure to delete this item ?  
Để người dùng lựa chọn và thực hiện ngay tại Home Screen 

Về việc ứng dụng MVVM thì a cũng k đồng ý với việc sử dụng 2 viewmodel trong trường hợp này 
Em có thực hiện nhiều hành động của các repository khác nhau trong 1 ViewModel. 
Bản chất ViewModel là chứa dữ liệu cho View chứ không phải là xử lý logic của View 
nên không cần phải tách ra và tái sử dụng HomeViewModel